plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeTransformTelemetryManager.kt

```diff
@@ -216,7 +216,7 @@ class CodeTransformTelemetryManager(private val project: Project) {
         } else if (connection.connectionType == ActiveConnectionType.BUILDER_ID && connection is ActiveConnection.ValidBearer) {
             authType = CredentialSourceId.AwsId
         }
-        CodetransformTelemetry.jobIsStartedFromChatPrompt(codeTransformSessionId = sessionId, credentialSourceId = authType)
```

Comment:
Curious: why did this get changed?

Reply:
this is because the CodeTransform team changed the event name in the newer version of `aws-toolkit-common`, but it is not updated in the IDE toolkit code yet, without changing this, the build would fail

VSCode equivalent of this PR also has the same change: https://github.com/aws/aws-toolkit-vscode/pull/5296/files

Reply:
The change to the metric was not by code transform team. We are adding back the metric that was removed. Please do not change the usage of this metric. https://github.com/aws/aws-toolkit-common/pull/788